services-manager/src/main/resources/application-debug.yaml

```diff
@@ -39,3 +39,12 @@ manager:
 
     user-service:
       level: *debug
+
+    attribute-masking-service:
+      level: *debug
```

Comment:
Would it better if these were in alphabetical order, rather than just tagged on the end?